utility/src/main/java/org/oppia/android/util/platformparameter/PlatformParameterConstants.kt

```diff
@@ -225,3 +225,13 @@ const val NPS_SURVEY_MINIMUM_AGGREGATE_LEARNING_TIME_IN_A_TOPIC_IN_MINUTES =
  * experience with the app.
  */
 const val NPS_SURVEY_MINIMUM_AGGREGATE_LEARNING_TIME_IN_A_TOPIC_IN_MINUTES_DEFAULT_VALUE = 5
+
```

Comment:
Please move these changes to the [FeatureFlagConstants.kt](https://github.com/oppia/oppia-android/blob/develop/utility/src/main/java/org/oppia/android/util/platformparameter/FeatureFlagConstants.kt) file since that is the approach we will be using going forward.

Reply:
Done

Reply:
This looks good now.